### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ihucos**
https://github.com/ihucos/plash

It is a container build and runtime engine that plays well with docker. It is flexible at can cover a lot also more exotic use cases: Building images inside a docker container. Running docker images outside of docker completly unprivileged. Run docker in plash without installing it. build docker images with plash buildfiles, which have more high level configuration management capabilities and can be extended with python. It's kind of a "glue" that can fit into gaps but could also be used tu simply build and run containers without docker if you want to.

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)

- The project submitted conforms to the quality standards outlined in the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)

- Go to the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) and read it again.

I did read it

**RULES (because you probably didn't read the guidelines):**

Links will be tested with [awesome_bot](https://github.com/dkhamsing/awesome_bot). Please make sure your URL to be listed is unique (do some search queries), otherwise put a link at the end of `README.md` file.
 
- FORMAT
  - Use the following format: `[Name](Link) - Description.`
  - `[Name]` should be the title of the project
  - `(Link)` should be pointing to a GitHub repository, not to a website
  - `Description.` should end with a full stop/period
  - For GitHub repositories, add the name of the author with a link to its GitHub account after the description, like `By [@author](GitHub account)`
- ORDER: The entries are in lexical order (English letter order) 
- LABEL (Optional): Add one of the following icons next to the project name like ```[Name](Link) :icon: - Description.```
  - :heavy_dollar_sign: indicates if one needs a paying subscription to use the project.
  - :construction: indicates that the project is a work in progress/beta release.
  - :skull: indicates that the project is obsolete, but still worth mentioning it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,6 +170,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 - [dockit](https://github.com/humblec/dockit) :skull: - Do docker actions and Deploy gluster containers! By [@humblec](https://github.com/humblec)
 - [Grafeas](https://github.com/Grafeas/Grafeas) - A common API for metadata about containers, from image and build details to security vulnerabilities. By [Grafeas](https://github.com/Grafeas)
 - [Longshoreman](https://github.com/longshoreman/longshoreman) :skull: - Longshoreman automates application deployment using Docker. Just create a Docker repository (or use a service), configure the cluster using AWS or Digital Ocean (or whatever you like) and deploy applications using a Heroku-like CLI tool. By [longshoreman](https://github.com/longshoreman)
+- [plash](https://github.com/longshoreman/longshoreman) - A container run and build engine - runs inside docker.
```
Comment (veggiemonk) [inline README.md]: I guess this is not the URL of the project ;-)

**Comment by ihucos** [comment]
>The url is not the url of the project. Can you fix that please ?
>Also, What made you choose the category Deployment and Infrastructure ? What do you think about container-composition ?
Both sloppy mistakes, fixed.

> I went to see the code and I'm still confused about how it works...

Every subcommand is a separate script inside `plashlib/libexec`. One subcommand builds on top of the other and the use of python helper libraries is kept relatively small. The advantage of this architecture is that more low level components (`plash mount`, `plash add-layer`, `plash map`) are exposed in an accessible way.
`plashlib/stdlib.py` are functions of the very lightweigth configuration management language (which can be extended).
`plashlib/eval.py` is the interpreter of the very minimal command line based configuration management language.
`plashlib/tests` are the unit tests, which are packaged and can be run with `plash test`

Yeah, something like that is something to write in the README.
Maybe you can tell more what confused you, I am "too deep in it" to see whats not obvious about the code :-)

> the fact that I need to install it with python is not that convenient. Is there a tool that can package it for multiple platforms ?

I agree and am hoping this is not a long term problem. I have some ideas with PyInstaller.

>Please, please, please, put the website of the project ( https://ihucos.github.io/plash) in the field dedicated for this purporse by the top (next to 'A container build and runtime system') on the github project page. It took me quiet a while to find it.

Done

I am really glad you liked the project, thank you. It's more work than I first thought and difficult to explain why its so awesome even to other tech geeks, but I am getting there. Feel free to ask questions about it and give more feedback for example about the README.md

**Comment by tdensmore** [comment]
This link does not work:

Documentation <https://ihucos.github.io/plash/#documentation>

   - Reference: http://plash.io

**Comment by ihucos** [comment]
Yep, I just bought the domain and somehow am having troubles.
Now it is working - but will stop mysteriously to work in ~ 10 mins

**Comment by ihucos** [comment]
Github somehow "forgets" that I added a custom domain after a while

File: README.md [hunk 2]
```diff
@@ -158,6 +158,8 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 - [rocker](https://github.com/grammarly/rocker) - Extended Dockerfile builder. Supports multiple FROMs, MOUNTS, templates, etc. by [grammarly](grammarly).
 - [Stacker](https://github.com/stacker/stacker-cli) - Docker Compose Templates. Stacker provides an abstraction layer over Docker Compose and a better DX (developer experience).
 - [Zodiac](https://github.com/CenturyLinkLabs/zodiac) :skull: - A lightweight tool for easy deployment and rollback of dockerized applications. By [@CenturyLinkLabs][CenturyLinkLabs]
+- [plash](https://github.com/ihucos/plash) - A container run and build engine - runs inside docker.
```
Comment (veggiemonk) [inline README.md]: Every projects is order alphabetically. It should be inserted here: https://github.com/veggiemonk/awesome-docker/pull/536/files#diff-04c6e90faac2675aa89e2176d2eec7d8R157

**Comment by ihucos** [comment]
@veggiemonk 
Done, thanks for guiding me through the process

@tdensmore 
As now stated at the github project page plash's documentation is now hosted at http://plash.io/

**Pull request merged**